src/highdicom/sr/templates.py

```diff
@@ -1837,7 +1854,8 @@ def __init__(
             )
         item = ContainerContentItem(
             name=title,
-            template_id='1500'
+            template_id='1500',
+            relationship_type=RelationshipTypeValues.CONTAINS
```

Comment:
I am not sure if this is correct in case of a root template, which per definition doesn't have a parent.

Reply:
See discussion [below](https://github.com/MGHComputationalPathology/highdicom/pull/80#discussion_r645236633)